src/services/shopify-client.ts

```diff
@@ -0,0 +1,25 @@
+export enum SubscriberListState {
+  ready,
+  synchronizing_contacts,
+}
+
+export interface SubscriberList {
+  name: string;
+  id: number;
+  amount_subscribers: number;
+  state: SubscriberListState;
+}
+
+export interface ShopifyConnection {
+  shop_name: string;
```

Comment:
What do you think about:

```suggestion
  shopName: string;
```

Reply:
We don't use underscores anywhere right?

Reply:
Mmm, no, in general, we are using camel case.